interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName[:len(baseName)-3]+".bpf")
```

Comment:
Thanks for bringing this up, this is still the case and a really important point. In order to provide reverts that work we will need to change the `dirs.SnapSeccompDir` in this PR. Otherwise the old profiles will get removed and we are back to the race where snapd is not quick enough to restore the profiles on system-boot. I'm open for suggestions, instead of '/var/lib/snapd/profiles` we could use `/var/lib/snapd/seccomp/bpf` or just `/var/lib/snapd/seccomp` or `/var/lib/snapd/security/seccomp` ?